ax/models/torch/botorch_modular/sebo.py

```diff
@@ -233,16 +232,34 @@ def optimize(
             candidates, a tensor with the associated acquisition values, and a tensor
             with the weight for each candidate.
         """
+        # Prepare arguments for optimizer
+        optimizer_options_with_defaults = optimizer_argparse(
+            self.acqf,
+            optimizer_options=optimizer_options,
+            optimizer="optimize_acqf_homotopy",
+        )
+
+        _tensorize = partial(torch.tensor, dtype=self.dtype, device=self.device)
+        ssd = search_space_digest
+        bounds = _tensorize(ssd.bounds).t()
+
+        optimizer_options["batch_initial_conditions"] = get_batch_initial_conditions(
+            acq_function=self.acqf,
+            raw_samples=optimizer_options_with_defaults["raw_samples"],
+            inequality_constraints=inequality_constraints,
+            fixed_features=fixed_features,
+            X_pareto=self.acqf.X_baseline,
+            target_point=self.target_point,
+            bounds=bounds,
+            num_restarts=optimizer_options_with_defaults["num_restarts"],
+        )
```

Comment:
The motivation to include pareto samples for initializing the L0 optimization in addition to the random samples is the same for the L1 case. It creates an unfair comparison between the L1 and L0 methods to have different init strategies. Per https://github.com/facebook/Ax/pull/2938#issuecomment-2541761382 happy to revert this if would rather keep the change more atomic

Reply:
Ah, I see. The two methods use different optimizer. It makes sense to keep the two consistent.

Reply:
will revert and then happy to fix/change in another PR

Reply:
> Ah, I see. The two methods use different optimizer. It makes sense to keep the two consistent.

sorry didn't see this reply before deciding it would be better to limit scope as otherwise more benchmarking might be needed. I have only tested the L0 implementation in my own work.